packages/npm/resx/OPIResources-1.0.0-alpha5.resx

```diff
@@ -0,0 +1,696 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is it your intention to check in this generated file to this repo?

Reply:
Good question. Maybe not. I'd like to leave it for now though.